Makefile

```
@@ -0,0 +1,21 @@
+# Makefile for TF1 HealthCheck project
+
+BINARY_NAME=healthcheck
+
+.PHONY: test build run clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)